tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -232,3 +235,26 @@ def test_nodereplacement_proactive(self):
         assert ceph_cluster_obj.wait_for_rebalance(
             timeout=1800
         ), "Data re-balance failed to complete"
+
+
+@tier4b
+@skipif_no_lso
+@bugzilla("1840539")
+class TestNodeReplacementLSO(ManageTest):
+    """
+    Node replacement twice on LSO:
+    node_x -> node_y
+    node_z -> node_x
+    """
+
+    def test_nodereplacenet_lso(self):
+        for i in range(2):
```

Comment:
Do we need to use in the current iteration the node from the previous iteration? If not, I think you can just use these 2 lines from the file https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/nodes/test_node_replacement_proactive.py: 

> osd_node_name = select_osd_node_name()
        delete_and_create_osd_node(osd_node_name)

If you use an LSO cluster to run the test the function `delete_and_create_osd_node(osd_node_name)` will trigger the LSO function `delete_and_create_osd_node_vsphere_upi_lso`.

Reply:
Right. Done